### PR TITLE
Issue comp 16

### PR DESCRIPTION
**Pull request by balmas**
for alpheios-project/components#16

See PR alpheios-project/components#740 for discussion

### CODE REVIEW

File: src/adapters/base-adapter.js [hunk 1]
```diff
@@ -155,19 +155,16 @@ class BaseAdapter {
...
-      console.info(error.response.data)
-      console.info(error.response.status)
-      console.info(error.response.headers)
+      console.warn('Unexpected response retrieving data from an Alpheios service', error)
     } else if (error.request) {
       // The request was made but no response was received
       // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
       // http.ClientRequest in node.js
-      console.info(error.request)
+      console.warn('No response from an Alpheios service', error)
```
Comment (irina060981) [inline src/adapters/base-adapter.js]: may be it should be console.error (by its sense)

File: src/adapters/base-adapter.js [hunk 2]
```diff
@@ -155,19 +155,16 @@ class BaseAdapter {
...
     } else {
       // Something happened in setting up the request that triggered an Error
-      console.info('Error', error.message)
+      console.warn('Unexpected error requesting data from an Alpheios service', error.message)
```
Comment (irina060981) [inline src/adapters/base-adapter.js]: the same -  may be it should be console.error (by its sense)

File: src/adapters/concordance/adapter.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ class AlpheiosConcordanceAdapter extends BaseAdapter {
         return {
           wordUsageExamples: parsedWordUsageList,
           targetWord: homonym.targetWord,
-          language: homonym.language,
+          language: LanguageModelFactory.getLanguageCodeFromId(homonym.languageID),
```
Comment (irina060981) [inline src/adapters/concordance/adapter.js]: Bridget, why did you change it to the same?
```
get language () {
    console.warn(`Please use languageID instead`)
    return LMF.getLanguageCodeFromId(this.languageID)
  }
```

File: src/adapters/concordance/adapter.js [hunk 4]
```diff
@@ -289,7 +289,7 @@ class AlpheiosConcordanceAdapter extends BaseAdapter {
   */
   createWordUsageExample (jsonObj, homonym, author, textWork, passage) {
     let source = this.config.sourceTextUrl + jsonObj.link
-    let wordUsageExample = new WordUsageExample(homonym.language, jsonObj.target, jsonObj.left, jsonObj.right, source, jsonObj.cit)
+    let wordUsageExample = new WordUsageExample(LanguageModelFactory.getLanguageCodeFromId(homonym.languageID), jsonObj.target, jsonObj.left, jsonObj.right, source, jsonObj.cit)
```
Comment (irina060981) [inline src/adapters/concordance/adapter.js]: the same question

File: src/adapters/base-adapter.js (hunk 1 above)
Comment (balmas) [reply src/adapters/base-adapter.js]: ok yes, will fix.

File: src/adapters/base-adapter.js (hunk 2 above)
Comment (balmas) [reply src/adapters/base-adapter.js]: agree

File: src/adapters/concordance/adapter.js (hunk 3 above)
Comment (balmas) [reply src/adapters/concordance/adapter.js]: because homonym.language is deprecated and throws a warning.
